sdk/maps/maps-render-rest/samples-dev/getMapStaticImage.ts

```diff
@@ -105,13 +105,13 @@ async function main(): Promise<void> {
 
   // Make the request
   const res3 = await client
-    .path("/map/static/{format}", "png")
+    .path("/map/static")
     .get({
       queryParameters: {
         bbox: [13.228, 52.4559, 13.5794, 52.62],
         zoom: 10,
-        path,
-        pins,
+        path: path,
+        pins: pins,
```

Comment:
Why do we need this change? `path,` is equal to `path: path,` so does `pins` and `pins: pins`.